src/main/java/com/kodilla/ecommercee/domain/Product.java

```diff
@@ -1,11 +1,15 @@
 package com.kodilla.ecommercee.domain;
 
-import javax.persistence.Entity;
-import javax.persistence.Id;
-import javax.persistence.JoinColumn;
-import javax.persistence.ManyToOne;
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
 
 @Entity
+@Getter
+@Setter
 public class Product {
     @Id
     private Long id;
```

Comment:
add @Column(name = "product_id")

Reply:
W sumie mogę dodać już not null i auto generated.

Reply:
Klasy Product nie chciałem rozwijać za bardzo, bo nie jestem odpowiedzialny za jej implementację.

Reply:
Tak, ale jak aplikacja się nie buduje to nie puści nam PR.